src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");
```

Comment:
The message 'Bulldozer started work' could be more informative. It should indicate that it is the Bulldozer machine that is working. Consider including the class name in the message.